packages/ember-debug/lib/index.js

```diff
@@ -16,19 +16,21 @@ export { default as Error } from './error';
 export { isTesting, setTesting } from './testing';
 
 // These are the default production build versions:
-let assert = () => {};
-let info = () => {};
-let warn = () => {};
-let debug = () => {};
-let deprecate = () => {};
-let debugSeal = () => {};
-let debugFreeze = () => {};
-let runInDebug = () => {};
+const noop = () => {};
```

Comment:
tried, its failed travis, (there was some weirdness to it I forgot), so reverted to initial per line approach